dataprofiler/profilers/float_column_profile.py

```diff
@@ -105,23 +105,39 @@ def _get_float_precision(cls, df_series):
         :return: string representing its precision print format
         :rtype: int
         """
-        integer_decimal_loc = -1
-        float_precision = None
+        if df_series.empty:
+            return 0
+
+        # set it to first value length, at very least will be smaller since min.
+        precision = len(df_series.iloc[0])
```

Comment:
what if `nan` is the first value?

Reply:
This function is created under the assumption that it is receiving a cleaned dataset, i.e. no `nan`s are provided to it.

The profilers should never receive a nan into its update profile as the series is cleaned prior to that point.

If we want to account for `nan`s it will make the function slower (maybe not much, unsure)? However, we would have to do that for all the profiles as all of them make the same assumption. If this is desired, maybe it should be in a separate PR cleaning up all profiles for this case.